app/services/smart-banner.js

```diff
@@ -30,8 +33,18 @@ export default Service.extend({
     'smartBannerAdConfiguration.text',
     'isInCustomSmartBannerCountry',
     'isSystemTargetedByCustomSmartBanner',
+    'willUapNotAppear',
   ),
 
+  init() {
```

Comment:
`39:3  error  Call this._super(...arguments) in init hook  ember/require-super-in-init`